### DISCUSSION TITLE
Making Action Events as Audit Trail for Nova

### DISCUSSION THREAD

**Discussion by cheehau** [laravel/nova-issues#5646] [Q&A] [answered]
Hi,
I'm not sure if there's any similar discussion else where, please close if duplicated.
I started to use Laravel Nova to build an ERP, where audit trail is important requirement. 
Initially i'm exploring https://laravel-auditing.com/, but later I found out that the action_events table recorded the changes as well, and there's a hidden resources/action-events that serves as audit trail.

I did some changes for my own project
https://cheehau.dev/notes/laravel-nova-action-events-audit-trail
However, it's such a waste of feature if it stay hidden.

I suggest to have something similar like `Nova::enableAuditTrail();` in NovaServiceProvider.

Let me know your thoughts.

**Comment by crynobone** [comment, marked as answer]
You can update `config/nova.php` and extends the default Action Resource and make changes (including allowing it to be displayed in navigation): 

```php
/**
 * Indicates if the resource should be displayed in the sidebar.
 *
 * @var bool
 */
public static $displayInNavigation = true;
```
